README.md

```diff
@@ -67,6 +71,40 @@ Under an automated build environment it makes sense to have a command return a n
 
 ```
 
+### Creating a secure VPN tunnel with ngrok
+
+The CLI has built-in support for [ngrok](https://ngrok.com/) to make it easier for you to run your tests against a locally-accessible application. In order to set up `ngrok` you will need your access token from [your ngrok account](https://dashboard.ngrok.com/get-started/your-authtoken). To initiate a tunnel on execution use the `--ngrokTunnel` parameter to specify your local endpoint, this can be a port on your local computer or a target on the local network:
+
+```
+❯ ghost-inspector test execute <testId> \
+  --ngrokTunnel localhost:8000 \
+  --ngrokToken '<my-ngrok-token>'
+```
+
+If you prefer you can also set the ngrok token using the environment variable `NGROK_TOKEN`.
```

Comment:
It's probably worth noting that `--immediate true` cannot be combined with `--ngrokTunnel` because it'll teardown ngrok as soon as the response comes back (I think?)

Reply:
Yeah, I just [throw an error](https://github.com/ghost-inspector/node-ghost-inspector/pull/34/files#diff-ad2b0fce270681f35fc87ef52afdbbceba9bf0c46feb1d7211f3029507527f8eR97) if `--ngrokTunnel` is combined with `--immediate`